src/electron/views/root-container/components/platform-body-class-modifier.tsx

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { BodyClassModifier } from 'common/components/body-class-modifier';
+import { DocumentManipulator } from 'common/document-manipulator';
+import { NamedFC } from 'common/react/named-fc';
+import { PlatformInfo } from 'electron/window-management/platform-info';
+import * as React from 'react';
+
+export type PlatformBodyClassModifierDeps = {
+    documentManipulator: DocumentManipulator;
+    platformInfo: PlatformInfo;
+};
+
+export type PlatformBodyClassModifierProps = {
+    deps: PlatformBodyClassModifierDeps;
+};
+
+export const PlatformBodyClassModifier = NamedFC<PlatformBodyClassModifierProps>(
+    'PlatformBodyClassModifier',
+    props => {
+        if (props.deps.platformInfo.isMac()) {
```

Comment:
does it make sense to reverse this conditional, make the clause guard the one that return null? Doing so makes the "then" block a one-liner which is always easier to read.